sale_order_revision/view/sale_order.xml

```diff
@@ -30,7 +30,7 @@
                     type="object"
                     class="oe_stat_button"
                     icon="fa-file-archive-o"
-                    attrs="{'invisible': ['|', ('has_old_revisions', '=', False),('revision_count', '=', 0)]}"
+                    invisible="has_old_revisions == False or revision_count == 0"
```

Comment:
Could also be written as 
`invisible="not has_old_revisions or revision_count == 0"`